client/wildcard/src/components/Tabs/Tabs.tsx

```diff
@@ -8,34 +8,46 @@ import {
     TabPanelsProps as ReachTabPanelsProps,
     TabProps as ReachTabProps,
     Tabs as ReachTabs,
-    TabsProps as ReachTabsProps,
+    TabsProps as ReachTabsProps
 } from '@reach/tabs'
 import classNames from 'classnames'
 import React from 'react'
 
-import { TabsContext, useTabsContext } from './context'
+import { TabPanelIndexContext, TabsSettingsContext, useTabsSettings } from './context'
 import styles from './Tabs.module.scss'
-import { useTabPanelBehavior } from './useTabPanelBehavior'
-import { useTabPanelsState } from './useTabPanelsState'
-import { TabsState, useTabs } from './useTabs'
+import { useShouldPanelRender } from './useShouldPanelRender'
 
-interface TabsProps extends ReachTabsProps, TabsState {
+export interface TabsSettings {
+    /* Tab component font size.
+     * Default is "small" */
+    size?: 'small' | 'medium' | 'large'
+    /* true: only load the initial tab when tab component mounts
```

Comment:
```suggestion
    /** Tab component font size.
     * Default is "small" */
    size?: 'small' | 'medium' | 'large'
    /** true: only load the initial tab when tab component mounts
```

So it is valid JSDoc and shows in hover definitions :)

Reply:
That's a great catch, thanks Tom!